ops/ccs-ops-misc/load-test/locustfile.py

```diff
@@ -0,0 +1,54 @@
+import csv
+import os
+import sys
+import urllib3
+from locust import HttpUser, task
+
+'''
+If there is a public key to verify the BFD Server's responses
+then it can be passed in with an environment variable. Otherwise,
+the error from the self-signed cert is ignored. The warnings are also
+disabled because thousands will appear in the logs and make it difficult
+to see anything else.
+'''
+try:
+    server_public_key = os.environ['SERVER_PUBLIC_KEY']
+except KeyError:
+    server_public_key = False
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+'''
+A file of beneficiary IDs to use in requests and a
+client cert to authenticate with BFD are both required.
+'''
+try:
+    ids_file = os.environ['IDS_FILE']
+    client_cert = os.environ['CLIENT_CERT']
+except KeyError as err:
+    print(f'must provide env var: {err}')
+    sys.exit(1)
+
+try:
+    with open(ids_file, 'r') as f:
+        eob_ids = f.read().split('\n')
+
+        # remove empty string at the end
+        eob_ids.pop()
+
+        patient_ids = eob_ids.copy()
+except FileNotFoundError as err:
+    print(f'key file not found: {err}')
+    sys.exit(1)
+
+after_date = '2021-04-26'
```

Comment:
should probably be a param, or maybe date now - n days, else as time goes by this could impact results dramatically

Reply:
This script will have to be modified for each test, so I'm not overly worried about that. I ran a test at first that didn't have any kind of date to simulate those expensive requests. 

But I can make it default to two weeks prior and add a note in the instructions, just so people don't need to solve the date stuff themselves.